src/ui/common/src/utils/workflows.tsx

```diff
@@ -26,6 +26,63 @@ export type RetentionPolicy = {
   k_latest_runs: number;
 };
 
+export type WorkflowMetrics = {
```

Comment:
should be able to use `common/src/handlers/responses/operator/OperatorResultResponse` here